elements/pfe-button/src/pfe-button.scss

```diff
@@ -0,0 +1,138 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: button;
+
+:host {
+  // Example local variable, using BEM format
+  // --pfe-button--Property: value;
+  display: inline-block;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-variant="primary"]) button,
+button {
+  background-color: #06c;
+  color: #fff;
+  font-size: 16px;
+  padding: 8px 16px;
```

Comment:
Feel like this should be em, since it should be proportionate to the font size.

Reply:
Prefer we use the pfe-var and pfe-color functions and pull from the theme instead of hardcoding values 👍

Reply:
@castastrophe I've updated as many variables as I could, but there were still a few I couldn't find or just didn't know how to use. For example, the red background color for the danger variant is `#a30000`, which maps to `$pf-color-red-200` in`_pf-colors.scss`, but I couldn't figure out how to pull it into the component 😞. Also, in `pfe-apply-broadcast(text)`, how can I specify that I want the fallback to be `#151515` instead of `#333`. I thought I could pass it as a second arg, but it screamed at me when I did.